### PR TITLE
Import Chiltern (closes #2348)

### LINKED ISSUE #2348: Import E07000005-Chiltern District Council
Xpress DC

### PR DESCRIPTION
**Pull request by alexsdutton**

### CODE REVIEW

File: polling_stations/apps/data_collection/management/commands/import_chiltern.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    def address_record_to_dict(self, record):
+        uprn = record.property_urn.strip().lstrip("0")
+
+        if record.addressline6 == "HP5 SPJ":
+            record = record._replace(addressline6="HP5 2PJ")
+
+        if uprn == "10013777238":
+            print(record)
+            record = record._replace(addressline6="HP16 0RL")
+            print(record)
```
Comment (chris48s) [inline polling_stations/apps/data_collection/management/commands/import_chiltern.py]: we can bin these print statements

File: polling_stations/apps/data_collection/management/commands/import_chiltern.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            "200003036501",  # HP169LQ -> HP160RR : Ballinger Meadow, Herberts Hole, Great Missenden
+            "10012939600",  # HP169BY -> HP169BG : St Martins, Grimms Hill, Great Missenden, Bucks
+            "100080498318",  # HP66RT -> HP66SE : The Grove, 19 Amersham Road, Little Chalfont
+            "100081080415",  # HP84EE -> HP84EF : Bow Wood Barn, Bottom House Lane, Chalfont St. Giles, Bucks
+            "200003030834",  # HP160LR -> HP160RL : Hawthorn Farm, Hyde End, Chesham Road, Great Missenden, Bucks
+        ]:
+            rec["accept_suggestion"] = True
+
+        if uprn in [
+            "10013777238",  # see above
```
Comment (chris48s) [inline polling_stations/apps/data_collection/management/commands/import_chiltern.py]: what's the thing above that I'm seeing?

Comment (alexsdutton) [reply polling_stations/apps/data_collection/management/commands/import_chiltern.py]: The postcode fix using the same uprn ;-). I've clarified the comment

File: polling_stations/apps/data_collection/management/commands/import_chiltern.py (hunk 1 above)
Comment (alexsdutton) [reply polling_stations/apps/data_collection/management/commands/import_chiltern.py]: Done!

**Pull request merged**